pilot/test/integration/BUILD

```diff
@@ -6,6 +6,7 @@ go_library(
         "access.go",
         "driver.go",
         "egress_rules.go",
+	"egress_rules_http_tls_origination.go",
```

Comment:
What’s this for?

Reply:
This is to tesl tls origination on http egress traffic, which we still plan to support